frontend/packages/multicluster-sdk/src/api/use-fleet-k8s-watch-resource/use-fleet-k8s-watch-resource.ts

```diff
@@ -36,7 +36,9 @@ export const useFleetK8sWatchResource = <R extends FleetK8sResourceCommon | Flee
 ): WatchK8sResult<R> | [undefined, boolean, any] => {
   const { cluster, ...resource } = initResource ?? {}
   const useFleet = cluster && cluster !== hubClusterName
-  const nullResource = !(resource?.groupVersionKind ?? resource)
+  const nullResource =
+    initResource === null || initResource === undefined || !initResource || !resource?.groupVersionKind
+
```

Comment:
```suggestion
  const nullResource = !initResource || !resource?.groupVersionKind
```
@upalatucci could we shorten to this?

Reply:
yes